roles/cifmw_cephadm/tasks/configure_object.yml

```diff
@@ -38,21 +38,97 @@
+      register: service_swift_uuid
+
+    - name: Get uuid for role 'member'
+      ansible.builtin.shell: "set -o pipefail && oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack role show member -c id -f value"
+      register: role_member_uuid
+
+    - name: Get uuid for role 'admin'
+      ansible.builtin.shell: "set -o pipefail && oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack role show admin -c id -f value"
+      register: role_admin_uuid
+
+    - name: Get uuid for project 'admin'
+      ansible.builtin.shell: "set -o pipefail && oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack project show admin -c id -f value"
+      register: project_admin_uuid
+
+    - name: Get uuid for user 'admin'
+      ansible.builtin.shell: "set -o pipefail && oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack user show admin -c id -f value"
+      register: user_admin_uuid
+
+    - name: Get uuid for role 'swiftoperator'
+      ansible.builtin.shell: "set -o pipefail && oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack role show swiftoperator -c id -f value"
+      register: role_swiftoperator_uuid
+
+- name: Check if cifmw_cephadm_certificate path exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_cephadm_certificate }}"
+  register: cert_stat_result
+  when: cifmw_cephadm_certificate | length > 0
+
+- name: Check if cifmw_cephadm_key path exists
+  ansible.builtin.stat:
+    path: "{{ cifmw_cephadm_key }}"
+  register: key_stat_result
+  when: cifmw_cephadm_key | length > 0
+
+- name: Update urischeme based on cert/key
+  ansible.builtin.set_fact:
+    cifmw_cephadm_urischeme: "https"
+  when:
+    - cifmw_cephadm_certificate | length > 0
+    - cifmw_cephadm_key | length > 0
+    - cert_stat_result.stat.exists | bool
+    - key_stat_result.stat.exists | bool
+
+- name: Configure object store to use rgw
+  cifmw.general.ci_script:
+    extra_args:
+      KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    output_dir: "/home/zuul/ci-framework-data/artifacts"
+    script: |-
+      oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack role add --user {{ user_swift_uuid.stdout }} --project {{ project_service_uuid.stdout }} {{ role_member_uuid.stdout }}
+      oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack role add --user {{ user_swift_uuid.stdout }} --project {{ project_service_uuid.stdout }} {{ role_admin_uuid.stdout }}
+      oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack endpoint create --region regionOne {{ service_swift_uuid.stdout }} public {{ cifmw_cephadm_urischeme }}://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
+      oc -n {{ cifmw_cephadm_ns }} rsh openstackclient openstack endpoint create --region regionOne {{ service_swift_uuid.stdout }} internal {{ cifmw_cephadm_urischeme }}://{{ cifmw_cephadm_vip }}:8080/swift/v1/AUTH_%\(tenant_id\)s
```

Comment:
For lines 129 and 130 I'll use two variables for the public and internal VIPs but I don't see how that affects `role add --user`.